src/test/kotlin/build/buf/gradle/ConfigurationTest.kt

```diff
@@ -28,23 +28,28 @@ class ConfigurationTest : AbstractBufIntegrationTest() {
         assertFailure()
     }
 
+    @Test
+    fun `project cannot use buf yaml workspaces and the protobuf gradle plugin first`() {
```

Comment:
For naming consistency with the other test cases, maybe "project cannot use buf-work-yaml and the protobuf-gradle-plugin, protobuf applied first"? Or does this new name miss the intent of this test case?